aws/rs_aws_rds/aws_rds_plugin.rb

```diff
@@ -584,7 +663,24 @@
     end
  
     output_path "//DBSecurityGroup"
- 
+	
+    output 'id' do
+     body_path 'OwnerId'
+    end
+
+    output 'name' do
+     body_path 'DBSecurityGroupName'
+    end
+
+    output 'region' do
+    end
+
+    output 'state' do
+    end
+
+    output 'tags' do
+    end
```

Comment:
do/end not needed if body_path isn't used.  Also if outputs without body_path can all be on one line.
output region, state, tags, ...